lib/src/se/llbit/chunky/renderer/RenderOptions.java

```diff
@@ -16,36 +16,49 @@
  */
 package se.llbit.chunky.renderer;
 
-public interface RenderConstants {
+/**
+ * The render options contain some values for tweaking the global render performance.
+ * <p><i>Which of these options a specific Renderer/RayTracer supports is not defined.
+ */
+public interface RenderOptions {
 
   /**
    * Default samples per pixel per pass
    */
   int SPP_PER_PASS_DEFAULT = 1;
 
+  int getSppPerPass();
+
+  /**
+   * Default CPU load
+   */
+  int TARGET_CPU_LOAD_PERCENTAGE = 100;
+
   /**
    * Default number of worker threads.
    * Is set to the number of available CPU cores.
    */
   int NUM_RENDER_THREADS_DEFAULT = Runtime.getRuntime().availableProcessors();
```

Comment:
As `NUM_RENDER_THREADS_MIN` changed to `RENDER_THREADS_COUNT_MIN`
should `NUM_RENDER_THREADS_DEFAULT` change to `RENDER_THREADS_COUNT_DEFAULT`?

or perhaps `RENDER_THREADS_DEFAULT_COUNT` and `RENDER_THREADS_MIN_COUNT`